packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewHostApiImpl.java

```diff
@@ -104,6 +108,35 @@ public View getView() {
     @Override
     public void dispose() {}
 
+    // TODO(bparrishMines): This should be removed once https://github.com/flutter/engine/pull/40771 makes it to stable.
+    // Temporary fix for https://github.com/flutter/flutter/issues/92165. The FlutterView is setting
+    // setImportantForAutofill(IMPORTANT_FOR_AUTOFILL_YES_EXCLUDE_DESCENDANTS) which prevents this
+    // view from automatically being traversed for autofill.
+    @Override
+    protected void onFocusChanged(boolean focused, int direction, Rect previouslyFocusedRect) {
+      super.onFocusChanged(focused, direction, previouslyFocusedRect);
+      if (focused && Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        final FlutterView flutterView = tryFindFlutterView();
+        if (flutterView != null) {
+          flutterView.setImportantForAutofill(IMPORTANT_FOR_AUTOFILL_YES);
+        }
+      }
+    }
```

Comment:
Alternative considered: 

According to autofill docs, a view can get around a parent view setting `IMPORTANT_FOR_AUTOFILL_YES_EXCLUDE_DESCENDANTS` if you do 

```java
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
  final AutofillManager manager = getContext().getSystemService(AutofillManager.class);
  manager.requestAutofill(webView);
}
```

However, this would require a call to `requestAutofill` every time a page finished loading. I tried to call to call it when the view was instantiated, but autofill didn't work. It only seemed to work if you call it once a page is loaded.